### DISCUSSION TITLE
Did I buy a License?

### DISCUSSION THREAD

**Discussion by proton32064** [aclap-dev/video-downloadhelper#1342] [License & Purchase requests]
### What's your issue?

I went through the check out process and the screen darkened but it never said it went through and I never got an email. Should I try again?

### The 10 first digits (only 10) of your license number or email address

Didn't get a License

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Normally, I would not respond here.  But @jcfrog Jérôme is in France & it may be a while before he can answer you.  In the meantime, I would NOT try again.  There's no sense in putting 2 charges on your credit card if your one attempt turns out to have worked.  You should check the web site of your credit card company (maybe that's actually your bank) to see if the charge is pending.  Even if it is not, I would still wait for Jérôme to respond before I did anything further.

**Comment by proton32064** [comment]
Thank you.  

    On Wednesday, October 9, 2024 at 10:19:38 PM EDT, GrampaWildWilly ***@***.***> wrote:  
 
 


I do not work for VDH. I'm just another user.

Normally, I would not respond here. But @jcfrog Jérôme is in France & it may be a while before he can answer you. In the meantime, I would NOT try again. There's no sense in putting 2 charges on your credit card if your one attempt turns out to have worked. You should check the web site of your credit card company (maybe that's actually your bank) to see if the charge is pending. Even if it is not, I would still wait for Jérôme to respond before I did anything further.

**Comment by jcfrog** [comment]
@proton32064 Hi, please send me your email address at licenses@downloadhelper.net
